ci/etc/expected_install_directories

```diff
@@ -53,5 +53,6 @@
 ./lib64/cmake/google_cloud_cpp_iam
 ./lib64/cmake/google_cloud_cpp_logging
 ./lib64/cmake/google_cloud_cpp_pubsublite
+./lib64/cmake/google_cloud_cpp_rest_internal
```

Comment:
`generate-libraries` seems to remove this for me.

Reply:
My naming is awful, as usual.  `ci/etc/expected_install_directories` only contains the directories created by automatically generated libraries.  `ci/cloudbuilds/builds/cmake-install.sh` contains the list of directories created by hand-crafted libraries.

Reply:
Meaning: we should move that hand-crafted directory to the `cmake-install.sh` script.